Doc/Manual/Php.html

```diff
@@ -73,18 +72,17 @@ <H1><a name="Php">32 SWIG and PHP</a></H1>
 available.
 </p>
 
-<H2><a name="Php_nn1">32.1 Generating PHP Extensions</a></H2>
+<H2><a name="Php_nn1">33.1 Generating PHP Extensions</a></H2>
 
 
 <p>
-To build a PHP extension, run swig using the <tt>-php</tt> option
-(you can also use <tt>-php7</tt> - PHP7 and PHP8 have a largely compatible C
-extension API, hence the same option name has been used for both).  For
-example:
+To build a PHP extension, run swig using the <tt>-php7</tt> option as follows
+(<tt>-php</tt> is also supported and currently is an alias for <tt>-php7</tt>
+but prior to SWIG 4.0.0 it was an alias for <tt>-php5</tt>):
 </p>
```

Comment:
This seems to be reverting changes from e3b112c69ceed5c39cb07fa45a3ba62b27712679.  There are more reverted changes below and there seem to be reverted changes in some other chapters too.

I'd suggest `git diff --ignore-matching-lines='<[Hh][123456]><[Aa]\>'` should return approximately empty output for all chapters of the manual except the new Fortran one, `Contents.html` and `Sections.html`.